### PR TITLE
Small cleanup and a subtest for foreign constraints

### PR DESCRIPTION
**Pull request by adizaimi**
Signed-off-by: Adi Zaimi <azaimi@bloomberg.net>

### CODE REVIEW

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: <b>Error</b>. <a href='http://comdb2.s3-website-us-east-1.amazonaws.com/formatting-a-pr.html'>Use clang-format</a> or <a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/25f1278/format.diff'>view diff</a> &#x26A0;.<br>Smoke testing: <a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/25f1278/smoketest.log'>Error &#x26A0; </a>.<br>Failed to compile.</p><p><a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/25f1278/detail.txt'>Detail</a> | <a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/25f1278/logs.tar.gz'>Log</a> | <a href='http://ec2-34-202-216-70.compute-1.amazonaws.com/restricted/index.php?id=2508'>Resubmit</a></p>

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: <b>Error</b>. <a href='http://comdb2.s3-website-us-east-1.amazonaws.com/formatting-a-pr.html'>Use clang-format</a> or <a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/835b97b/format.diff'>view diff</a> &#x26A0;.<br>Smoke testing: Success &#10003;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: Success &#10003;.</p><p>The first 10 failing tests are: <br>socksql_master_swings [core dumped]</p><p><a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/835b97b/detail.txt'>Detail</a> | <a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/835b97b/logs.tar.gz'>Log</a> | <a href='http://ec2-34-202-216-70.compute-1.amazonaws.com/restricted/index.php?id=2508'>Resubmit</a></p>

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: <b>Error</b>. <a href='http://comdb2.s3-website-us-east-1.amazonaws.com/formatting-a-pr.html'>Use clang-format</a> or <a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/33dcec2/format.diff'>view diff</a> &#x26A0;.<br>Smoke testing: Success &#10003;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: Success &#10003;.</p><p><a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/33dcec2/detail.txt'>Detail</a> | <a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/33dcec2/logs.tar.gz'>Log</a> | <a href='http://ec2-34-202-216-70.compute-1.amazonaws.com/restricted/index.php?id=2508'>Resubmit</a></p>

File: tests/ddl_no_csc2.test/t11_cascade_drop.expected [hunk 1]
```diff
@@ -0,0 +1,24 @@
+(name='ddl_cascade_drop', description='On DROP, also drop the dependent keys/constraints. (Default: 1)', type='BOOLEAN', value='ON', read_only='Y')
```
Comment (nirbhayc) [inline tests/ddl_no_csc2.test/t11_cascade_drop.expected:1]: We also have a similar test here : https://github.com/bloomberg/comdb2/blob/c6b3b2ef685c24ca0a121bf28037da1be296011f/tests/ddl_no_csc2.test/t01_alter_table.sql#L119

**Review by nirbhayc — APPROVED** [review]
Good to merge.

File: tests/ddl_no_csc2.test/t11_cascade_drop.expected (hunk 1 above)
Comment (adizaimi) [reply tests/ddl_no_csc2.test/t11_cascade_drop.expected:1]: oh, nice. no harm having it twice i guess.
Thanks

**Comment by adizaimi** [comment]
Thank you Nirbhay!

**Pull request merged**